test/tools/runner/hooks/configuration.js

```diff
@@ -66,15 +66,14 @@ async function initializeFilters(client) {
 }
 
 const testSkipBeforeEachHook = async function () {
-  // `metadata` always exists, `requires` is optional
-  const requires = this.currentTest.metadata.requires;
+  const metadata = this.currentTest.metadata;
```

Comment:
Is there a reason for this change or it is purely cosmetic?

Reply:
When I first started trying to test this I had a more simulated approach that would pass sometimes so I tried using mocha's retries feature. It looks like the retried test doesn't get the metadata attached, so this crashed. I can revert, but we can't use retries with a metadata test currently.

Reply:
Ah makes sense.  Sounds good